app/src/main/res/layout/walkthrough_topic_summary_view.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.walkthrough.topiclist.topiclistviewmodel.WalkthroughTopicSummaryViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/walkthrough_topic_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:clickable="true"
+    android:elevation="4dp"
+    android:layout_margin="@dimen/padding_20"
+    android:focusable="true"
+    android:onClick="@{(v) -> viewModel.clickOnSummaryTile()}"
+    app:roundedRectDrawableWithColor="@{viewModel.backgroundColor}">
+
+    <ImageView
+      android:id="@+id/walkthrough_topic_thumbnail_image_view"
+      android:layout_width="match_parent"
+      android:layout_height="92dp"
+      android:importantForAccessibility="no"
+      android:scaleType="centerInside"
+      android:src="@{viewModel.topicSummary.topicThumbnail.thumbnailGraphic}"
+      app:layout_constraintBottom_toTopOf="@+id/bottom_container"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <FrameLayout
+      android:id="@+id/bottom_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintStart_toStartOf="parent">
+
+      <FrameLayout
+        android:id="@+id/bottom_topic_bar"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:bottomRoundedRectDrawableWithColor="@{viewModel.darkerBackgroundOverlayColor}"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/constraintLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="8dp">
+
+        <TextView
+          android:id="@+id/walkthrough_topic_name_text_view"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:ellipsize="end"
+          android:fontFamily="sans-serif-medium"
+          android:gravity="top"
+          android:maxLines="2"
+          android:layout_marginBottom="6dp"
+          android:minLines="2"
+          android:text="@{viewModel.name}"
+          android:textColor="@color/white"
+          android:textSize="14sp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
```

Comment:
Issue in UI:
Lesson count is behind the topic name

![Screenshot_1584704039](https://user-images.githubusercontent.com/9396084/77160313-17e5d680-6acd-11ea-8c2d-2fa901f6514b.png)